### ISSUE TITLE
Easier to retrieve Data Files for HandsOnII

### ISSUE THREAD

**Issue by Mixpap** [Santiastro1/Athens_School_2022#2]
We can add gdown in the dependencies (pip package in order for easier data downloading of drive data files.  Then it is easy:
`gdown --folder https://drive.google.com/drive/folders/1fxBK3qycT1fFmouz9JmIMw1mkAznfv1v`

`gdown --folder https://drive.google.com/drive/folders/1wfsr7_7A77ZuRE91m6dKHjtultoBN8Ef`

I had to do it because I work with a server and I cannot download locally, so it is more an enchantment than an issue 🙂

PS: I will probably spam you with "issues" @Santiastro1 🙂. I think this is the most convenient way if it is also ok by you.

**Comment by Santiastro1** [comment]
Yes of course! I will try to answer you as fast as possible so we can end up with the best environment for the students!

Thank you for taking care of revising it!
